sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts

```diff
@@ -202,6 +215,13 @@ function createDependencyData(span: ReadableSpan): RemoteDependencyData {
   const rpcSystem = span.attributes[SemanticAttributes.RPC_SYSTEM];
   // HTTP Dependency
   if (httpMethod) {
+    const httpUrl = span.attributes[SemanticAttributes.HTTP_URL];
```

Comment:
Accessing dictionary attributes won't through an exception if it doesn't exist in JS?

Reply:
This will only throw if span.attributes doesn't exist, in this case if HTTP_URL is not there will return undefined